src/Sulu/Bundle/ContactBundle/Domain/Event/AccountContactCreatedEvent.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Domain\Event;
+
+use Sulu\Bundle\ContactBundle\Admin\ContactAdmin;
+use Sulu\Bundle\ContactBundle\Entity\AccountContact;
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+
+class AccountContactCreatedEvent extends DomainEvent
```

Comment:
Would rename this event to `AccountContactAddedEvent` to be consistent with e.g. `AccountMediaAddedEvent`

Reply:
would use the `Account` entity as resource for all events 🙂